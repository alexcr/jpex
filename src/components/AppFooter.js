import React from 'react';
import styled from 'styled-components';

const Footer = styled.footer`
  margin-top: 30px;
  font-size: 12px;
  color: #333;
`;

export function AppFooter() {
  return (
    <Footer>
      Â© 2020 Alexandre Rondon
    </Footer>
  );
}
