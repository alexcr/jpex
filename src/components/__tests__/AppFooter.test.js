import React from 'react';
import { shallow } from 'enzyme';
import { AppFooter } from '../AppFooter';

describe('AppFooter', () => {
  it('should render footer correctly', () => {
    expect(
      shallow(<AppFooter />)
    ).toMatchSnapshot();
  });
});
