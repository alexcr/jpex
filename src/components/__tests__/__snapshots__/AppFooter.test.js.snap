// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppFooter should render footer correctly 1`] = `
<styled.footer>
  Â© 2020 Alexandre Rondon
</styled.footer>
`;
